src/bin/stratisd.rs

```diff
@@ -75,15 +69,21 @@ fn main() {
         .get_matches();
 
     if matches.is_present("debug") {
-        unsafe { DEBUG = true }
-    };
+        let mut builder = LogBuilder::new();
```

Comment:
I think ```filter()``` and ```parse()``` must be redundant.

Why not:

```
let mut builder = LogBuilder::new();
builder.format(|record: &LogRecord| format!("{} - {}", record.level(), record.args()));
if matches.is_present("debug") {
    builder.filter(Option("stratisd"), LogLevelFilter::Debug);
    builder.filter(Option("libstratis"), LogLevelFilter::Debug);
} else {
    builder.filter(Option("stratisd"), LogLevelFilter::Error);
    builder.filter(Option("libstratis"), LogLevelFilter::Error);
};
builder.init().unwrap();
```

I think we want to know about our, still, hypothetical, error messages even if we haven't passed the ```--debug``` flag. And we want our format on log messages to be uniform, regardless.